docs/source/usage/index.rst

```diff
@@ -284,3 +284,11 @@ Trunk Ports
 .. _Ironic boot-from-volume documentation: https://docs.openstack.org/ironic/latest/admin/boot-from-volume.html
 .. _python-esiclient: https://github.com/CCI-MOC/python-esiclient
 .. _python-esileapclient: https://github.com/CCI-MOC/python-esileapclient
+
+Serial Consoles
+~~~~~~~~~~~~~~~
```

Comment:
This should be '-------' instead to create a new subsection

Reply:
the other subsections all use '~~~~~~' though?

Reply:
The subsections under "Additional ESI CLI Actions" do, but this isn't an additional ESI CLI action.